deselected_tests.yaml

```diff
@@ -54,6 +54,10 @@ deselected_tests:
   # TODO: investigate copy failure of read-only buffer
   - linear_model/tests/test_coordinate_descent.py::test_read_only_buffer
 
+  # LinearRegression
```

Comment:
Reason of deselection?

Reply:
Difference is overflowed just a little bit

Reply:
I mean it should be added to file

Reply:
Fixed